Jenkinsfile

```diff
@@ -0,0 +1,13 @@
+buildMvn {
+  publishModDescriptor = 'no'
+  mvnDeploy = 'yes'
+  publishAPI = 'yes'
+  runLintRamlCop = 'yes'
+
+  doDocker = {
+    buildJavaDocker {
+      publishMaster = 'yes'
+      healthChk = 'yes'
+    }
+  }
+}
```

Comment:
@natali-zaitseva : Lets add the healthChkCmd back in here except that we'll use a different endpoint like the below:
`healthChkCmd = 'curl -sS --fail -o /dev/null http://localhost:8081/apidocs/ || exit 1'`. Per https://github.com/folio-org/raml-module-builder, "The maven-resources-plugin in pom.xml will copy the RAML files into a directory under /apidocs so that the runtime framework can pick it up and display html documentation based on the RAML files." So, this endpoint can work as our healthChkCmd. What do you think?